### ISSUE TITLE
Unexpected DASH DVB 2018 conformance errors 

### ISSUE THREAD

**Issue by kjerbi** [Dash-Industry-Forum/DASH-IF-Conformance#677]
Dear project maintainer,

I am testing dash DVB DASH 2018 conformance and found out some errors that we I don't understand:
1- error raised on content protection while our manifest does not have DRM protection at all.
2- error related to DVB profile while the profile exists
3- error on Role for Audio adaptation set while the role exists.
4- Schematron raises errors when adaptation set and representation have their attributes not in alphabetic order. Is that mandatory or could it be just a warning?

Can you please help?

Many thanks in advanced :)

Here is my manifest (with alphabetic order fixed):

[manifest_dvb2018.txt](https://github.com/Dash-Industry-Forum/DASH-IF-Conformance/files/12131149/manifest_dvb2018.txt)

**Comment by dsilhavy** [comment]
@kjerbi Thanks for reporting, we addressed issues 1-3 in #678 . Can you please check on https://staging.conformance.dashif.org/

**Comment by kjerbi** [comment]
@dsilhavy 
Thanks for the impressively quick fix!
I confirm that the reported issues are now solved. That is said, I wanted to solve one of the warnings regarding utc timing. I added the line <UTCTiming schemeIdUri="urn:mpeg:dash:utc:direct:2014" value="2023-07-21T15:13:29.342Z" />  right before the period as my packager does and I get Schematron error and the warning about utctiming persists.
Shall I open another ticket as the main issues of this ticket are already solved?
Many thanks again!
